browsers/deno.json

```diff
@@ -289,9 +289,23 @@
         "1.40": {
           "release_date": "2024-01-25",
           "release_notes": "https://github.com/denoland/deno/releases/tag/v1.40.0",
-          "status": "current",
+          "status": "retired",
           "engine": "V8",
           "engine_version": "12.0"
+        },
+        "1.41": {
+          "release_date": "2024-02-22",
+          "release_notes": "https://deno.com/blog/v1.41",
+          "status": "retired",
+          "engine": "V8",
+          "engine_version": "12.1"
+        },
+        "1.42": {
+          "release_date": "2024-03-28",
+          "release_notes": "https://deno.com/blog/v1.42",
+          "status": "current",
+          "engine": "V8",
+          "engine_version": "12.3"
```

Comment:
Note that Deno 1.41.3 already was on V8 12.3.

Reply:
@zloirock this was somewhat of a mistake and it was intended to land in v1.42.